src/tasks/che.ts

```diff
@@ -464,4 +464,86 @@ export class CheTasks {
         }
       }]
   }
+
+  verifyCheNamespaceExistsTask(flags: any, command: Command): ReadonlyArray<Listr.ListrTask> {
+    return [{
+      title: `Verify if namespace ${flags.chenamespace} exists`,
+      task: async () => {
+        if (!await this.che.cheNamespaceExist(flags.chenamespace)) {
+          command.error(`E_BAD_NS - Namespace does not exist.\nThe Kubernetes Namespace "${flags.chenamespace}" doesn't exist. The configuration cannot be injected.\nFix with: verify the namespace where Che workspace is running (kubectl get --all-namespaces deployment | grep workspace)`, { code: 'EBADNS' })
```

Comment:
`E_BAD_NS - Namespace` - I think colon is more commonly used instead of dash, but up to you...